### ISSUE TITLE
Want to change "CS_OPTION" id

### ISSUE THREAD

**Issue by kwayyinfotech** [Codestar/codestar-framework#384]
Hello,

I read this discussion:
https://github.com/Codestar/codestar-framework/issues/163

I have same requirements but I don't want to edit Codestar file.

Let me describe all the things:

I want to use Codestar Framework as plugin. So installed Codestar Framework as plugin. Than I created "cs-framework-override" folder in my theme and override all things. Now I want to change CS_OPTION id but I cannot modify it via my theme. I want to use Codestar Framework plugin in multiple themes so I want to keep it same as origial Github copy. So how to modify it via theme directly?

**Comment by Codestar** [comment]
Hi,

Yes, only you need to add this code, before include framework.

```php
define( 'CS_OPTION', 'mytheme_options' );
```

You do not need to override. because i am checking this define here: [cs-framework-path.php#L11](https://github.com/Codestar/codestar-framework/blob/master/cs-framework-path.php#L11)

```php
// See the code, there is checking if defined.
defined( 'CS_OPTION' ) or define( 'CS_OPTION', '_cs_options' );
```

**Comment by kwayyinfotech** [comment]
Yes I checked that code but still not working. Here is my code in functions.php file:

```

defined( 'CS_OPTION' ) or define( 'CS_OPTION', 'mytheme_options' );
function mytheme_function(){
	defined( 'CS_OPTION' ) or define( 'CS_OPTION', 'mytheme_options' );
}
add_action( 'init', 'mytheme_function' );
add_action( 'admin_init', 'mytheme_function' );
add_action( 'after_setup_theme', 'mytheme_function' );
```

**Comment by Codestar** [comment]
Try this.

```php
define( 'CS_OPTION', 'mytheme_options' );

// include framework
require_once get_template_directory() .'/cs-framework/cs-framework.php';
```

Still not working ?

**Comment by kwayyinfotech** [comment]
This is working. But I want to use cs framework purely as plugin so I cannot add files in my theme.

**Comment by Codestar** [comment]
Oh `as plugin` i understand now... yes, you are right there is an issue for change option name. let me think an idea about this.

**Comment by Codestar** [comment]
@kwayyinfotech can you leave an email via routewp[at]gmail[dot]com

**Comment by kwayyinfotech** [comment]
@Codestar  Any update on this? I am replying too late but now back with this point. Also I sent you email too.
